### ISSUE TITLE
Feature Request: Automate Spam detection WordPress.com Forums

### ISSUE THREAD

**Issue by nerdysandy** [Automattic/wp-calypso#97504]
### What

Of late, we have noticed an increase in Spam posts and comments on https://wordpress.com/forums/. The current process is for Forums Volunteers to tag something as `Spam` to be then manually checked by an HE to mark it as Spam. Since the count of spam has gone up and this involves manual process,  and there is wait time before HEs can get to this as we have other high priority tickets to handle first like sites being down etc., the spam continues to stay on the forums longer than before.

It will be helpful to scan the forums automatically to detect spam posts, comments, threads and close them to save both Volunteer and HE time and keep our forums clean.

### Why

As shared above this will streamline the process to keep our forums free of spam.

### How

A potential solution is already being discussed on this P2 but this had to be paused. I think that's a good solution to tackle spam.
> pfnqB3-bu-p2

**Comment by westi** [comment]
@nerdysandy The forums already have Akismet enabled

**Comment by nerdysandy** [comment]
Yeah, but the spam is still be to be closed/marked as spam manually when volunteers highlight it. The solution proposed in that P2 was to add a button to run spam check and mark the spam automatically without HE intervention.

**Comment by tashabishop02** [comment]
Reached out to @westi for help in getting updates for moving this forward.

**Comment by procifer** [comment]
FYI I'm doing some digging into the recent missed spam reports now.  I'm making some adjustments that hopefully will help with accuracy going forward.

**Comment by westi** [comment]
I dug in deeper on this and Akismet is running on the forums as expected, we caught some spam, missed some spam and are working on tweaking things based on the missed spam.

In general there is always going to be some manual work for spam because any automated system needs a human in the loop.

The referenced project is about introducing rescan functionality to help primarily with giant missed spam waves and when you first enable Akismet on a bbPress forum and want to clear up old spam.

**Comment by nerdysandy** [comment]
Is there a way to run this scan on any Forum threads which are tagged as `Spam` or `Spammer` by forum volunteers?

**Comment by joashrajin** [comment]
We've been getting hit with a lot of spam too on our mobile forums ([Simplenote](https://forums.simplenote.com/), [Day One](forums.dayoneapp.com/) and [Pocket Casts](https://forums.pocketcasts.com/))

They share the similar setup as the WordPress.com forums
